packages/shared/forks/ReactFeatureFlags.native-oss.js

```diff
@@ -82,7 +82,10 @@ export const enableAsyncActions = false;
 
 export const alwaysThrottleRetries = false;
 
-export const useMicrotasksForSchedulingInFabric = false;
```

Comment:
This flag is only set in `ReactFabricFiberConfig`, can we remove the feature flag and move this check there instead? Would be a small codesize win for paper, and remove another feature flag for us to maintain.

Reply:
We need to keep the feature flag because we have apps on bridgeless that can't use the event loop yet at Meta (so we can keep it dynamic in the `.fb.js` version).

Reply:
Ah, ok that makes sense.